tests/Integration/PrestaShopBundle/Command/GenerateMailTemplatesCommandTest.php

```diff
@@ -48,7 +48,7 @@ public function setUp(): void
 
     public function testMissingArguments()
     {
-        $this->expectException(RuntimeException::class, 'Not enough arguments (missing: "theme, locale").');
+        $this->expectException(RuntimeException::class);
```

Comment:
```suggestion
        $this->expectException(RuntimeException::class);
        $this->expectExceptionMessage('Not enough arguments (missing: "theme, locale").');
```
I would think this was the code that was supposed to be run.